rqt_topic/src/rqt_topic/topic_info.py

```diff
@@ -117,8 +118,10 @@ def get_bw(self):
             total = sum(self.sizes)
             bytes_per_s = total / (current_time - self.timestamps[0])
             mean_size = total / len(self.timestamps)
-            max_size = max(self.sizes)
-            min_size = min(self.sizes)
+            max_size = min_size = None
+            if self.sizes:
+                max_size = max(self.sizes)
```

Comment:
So with this and the next lines now returns `None` when `self.sizes` is not available, but the `{max, min}_size` are [not used](https://github.com/ros-visualization/rqt_common_plugins/blob/2b870bc05225b6a06582b391e300497997f44850/rqt_topic/src/rqt_topic/topic_widget.py#L204) so far.